site/content/videos.md

```diff
@@ -33,3 +33,5 @@ groups:
         link: /videos-list/
         desc: All of the Official ZAP videos, tagged and searchable
 ---
+### Latest Video
+{{<youtube uuid="s8DKKTTHsGM" small="true">}}
```

Comment:
Hmmm can we out a numeric id on them or pick by date instead of having to set it manually?

(Not a blocker)

Reply:
Good thinking - we could pick the first one in https://github.com/zaproxy/zaproxy-website/blob/main/site/data/videos.yaml - let me try that...

Reply:
Now renders the latest youtube vid 😁